platform.mk

```diff
@@ -90,6 +90,10 @@ PRODUCT_PACKAGES += \
 PRODUCT_PROPERTY_OVERRIDES += \
     ro.opengles.version=196610
 
+# Fluence
+PRODUCT_PROPERTY_OVERRIDES += \
+    ro.qc.sdk.audio.fluencetype=fluence
```

Comment:
With this addition, fluence is enabled on all supported platforms. Maybe you'd consider adding fluence to common.mk instead of platform.mk, and then remove from all other platforms in order to commonized this

Reply:
Is fluence supported in stock ? Just curious

Reply:
it is disabled on stock afaik, but haven't checked the 8.0 update to know for sure

Reply:
@alviteri
No, it is not enabled!
And not sure if fluence is not enabled in stock due to audio calibration mapping.
For sure that @richardtullberg is using some audio calibration tool that is capable to get those acdb ids and fix the fluence on SODP.

Reply:
We don't use Fluence on Xperia, I can't tell anyone what we do but we don't need Fluence to exist for it to understand that we are using dual mic.

The only thing I am worried about is if we have a platform where we do not have Dual Mics. But that might just be silly of me but its not impossible. Maybe we can set fluence= none in that specific platform.mk file just like we set it here to fluence.

@jerpelea What do you think? can we set fluence to enabled in common isntead of each platform?